http/vibe/http/server.d

```diff
@@ -530,20 +530,18 @@ alias HTTPServerErrorPageHandler = void delegate(HTTPServerRequest req, HTTPServ
 	will also drain the `HTTPServerRequest.bodyReader` stream whenever a request
 	body with form or JSON data is encountered.
 */
-enum HTTPServerOption {
-	none                      = 0,
-	/// Fills the `.path` and `.queryString` fields in the request
-	parseURL                  = 1<<0,
-	/// Deprecated: Fills the `.query` field in the request
-	parseQueryString          = 1<<1 | parseURL,
-	/// Deprecated: Fills the `.form` field in the request
-	parseFormBody             = 1<<2,
-	/// Deprecated: Fills the `.json` field in the request
-	parseJsonBody             = 1<<3,
-	/// Deprecated: Fills the `.files` field of the request for "multipart/mixed" requests
-	parseMultiPartBody        = 1<<4,
-	/// Deprecated: Fills the `.cookies` field in the request
-	parseCookies              = 1<<5,
+struct HTTPServerOption {
+	static enum none                      = 0;
+	deprecated("It is done lazy.")
```

Comment:
Maybe change these messages to `this is done lazily now`

Reply:
Plus this whole block of new lower case variants shouldn't exist at all because the upper case variants are being deprecated